src/main/java/club/bayview/smoothieweb/models/Problem.java

```diff
@@ -131,8 +131,8 @@ public ProblemLimits getProblemLimitForLang(String language) {
                 .setProblemId(id)
                 .setTestDataHash(hash)
                 .setGrader(SmoothieRunner.ProblemGrader.newBuilder()
-                        .setType(graderType.equals("") ? "endtrim" : graderType)
-                        .build())
+                .setType(graderType.equals("") ? "endtrim" : graderType)
```

Comment:
We actually want to keep the indent, because the extra indent shows that it's operating on the ProblemGrader "object builder", not the Problem "object builder"

Reply:
This issue has been fixed.